efel/pyfeatures/pyfeatures.py

```diff
@@ -51,6 +53,35 @@ def time():
     return _get_cpp_feature("time")
 
 
+def impedance():
+    from scipy.ndimage.filters import gaussian_filter1d
+    voltage_trace = voltage()
+    holding_voltage = _get_cpp_feature("voltage_base")
+    normalized_voltage = voltage_trace - holding_voltage
+    current_trace = current()
+    if current_trace is not None:
+        current_base = numpy.median(current_trace[0:10])
```

Comment:
so we still need a current_base feature for this, the 0:10 should not be hardcoded

Reply:
Yes and also the "median" option for both voltage_base and current_base features.

Reply:
Now we can use the current base feature with median but how can we set the range [0:10] there?

Reply:
@mariarv should this be 0:10? Wouldn't it be better to use the same range as voltage_base?

Reply:
I started using current_base in latest commit